packages/vscode-extension/src/accountTree.ts

```diff
@@ -312,6 +312,16 @@ export async function registerAccountTreeHandler(): Promise<Result<Void, FxError
           treeItem.label += " ";
           tools.treeProvider?.refresh([treeItem]);
         }
+      } else if (accountInfo !== undefined) {
+        const treeItem = {
+          commandId: "fx-extension.signinM365",
+          label: (accountInfo.upn as string) ? (accountInfo.upn as string) : "",
+          callback: signinM365Callback,
+          parent: TreeCategory.Account,
+          contextValue: "signedinM365",
+          icon: "M365",
+        };
+        tools.treeProvider?.add([treeItem]);
```

Comment:
I think here should be 'refresh()'?

Reply:
BTW, what about showing a warning to remind user he is offline, or user might get confused the account shows normally but error occurs

Reply:
I have tried with ```refresh```, the account info will not show.

![image](https://user-images.githubusercontent.com/13211513/149737837-dcad7927-fb37-4202-ac44-dc747b708ce4.png)

![image](https://user-images.githubusercontent.com/13211513/149737936-3d97eb9d-2798-4841-bd35-2ba64d700f65.png)

Reply:
It's weird, from the code logic, add() should be the one that doesn't work.

Reply:
I got it, the item is added the first time extension started. The tree view also listened to SharepointLoginProvider, will spfx deploy affect the status?

Reply:
I think so, spfx login can also trigger treeView update user info.

Reply:
Then is it needed to also update the sharepointLoginProvider getStatus() logic ?

Reply:
I update sharepoint logic, but it does not work. The treeProvider has not initialized when offline at this time.

![image](https://user-images.githubusercontent.com/13211513/149867464-ae5ea721-9ff1-4eac-9ab3-cc03048da1ac.png)

Reply:
Move setStatus location to make sure notify will update treeView.